schemas/telemetry/_common-schema.json

```diff
@@ -4,20 +4,27 @@
   "title": "CommonTelemetryProperties",
   "type": "object",
   "description": "Schema of common properties of Telemetry events",
-  "required": ["_dd", "date", "service", "version"],
+  "required": ["_dd", "type", "date", "service", "source", "version"],
   "properties": {
     "_dd": {
       "type": "object",
       "description": "Internal properties",
-      "required": ["event_type"],
+      "required": ["format_version"],
       "properties": {
-        "event_type": {
-          "type": "string",
-          "const": "internal_telemetry",
-          "description": "Event type"
+        "format_version": {
+          "type": "integer",
+          "const": 2,
+          "description": "Version of the RUM event format",
+          "readOnly": true
         }
       }
     },
+    "type": {
+      "type": "string",
+      "description": "RUM event type. Should specify telemetry only.",
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
      "description": "Telemetry event type. Should specify telemetry only.",
```
Those schemas are about "Telemetry events", versus schemas in the `schemas/rum` folder where we talk about "RUM events"